ui_framework/src/services/index.js

```diff
@@ -6,6 +10,5 @@ export {
 } from './accessibility';
```

Comment:
Whats the difference between ENTER_KEY from accessibility and keycodes.ENTER?  Above you are importing `import { keycodes, ENTER_KEY } from 'ui_framework/services';` which threw me off a bit. Seems like instead of ENTER_KEY it should just be keywords.ENTER